### PR TITLE
Add string conversion for client event's data property

### PR DESCRIPTION
**Pull request by tkqubo**
According to the pusher documentation ([Pusher Protocol](https://pusher.com/docs/pusher_protocol#events)), client event's data property needs to be a string type.  

> In order to keep the protocol consistent, Pusher tries to send the data field as a string.

This PR will provide that conversion logic into `PusherJsonSupport.scala`

Could you review this?

### CODE REVIEW

**Comment by dtaniwaki** [comment]
Yes, you are right! Thank you!

File: src/main/scala/com/github/dtaniwaki/akka_pusher/PusherJsonSupport.scala [hunk 1]
```diff
@@ -32,14 +32,15 @@ trait PusherJsonSupport extends DefaultJsonProtocol {
         "channel" -> JsString(event.channel),
         "user_id" -> JsString(event.userId),
         "event" -> JsString(event.event),
-        "data" -> event.data.toJson,
+        "data" -> JsString(event.data.toJson.toString),
```
Comment (dtaniwaki) [inline src/main/scala/com/github/dtaniwaki/akka_pusher/PusherJsonSupport.scala]: `compactPrint` is slightly better?

File: src/main/scala/com/github/dtaniwaki/akka_pusher/PusherJsonSupport.scala [hunk 2]
```diff
@@ -78,14 +79,14 @@ trait PusherJsonSupport extends DefaultJsonProtocol {
         case event: ClientEvent => event.toJson
       }.toVector
       JsObject(
-        "time_ms" -> JsNumber((res.timeMs.getMillis / 1000).toLong),
+        "time_ms" -> JsNumber(res.timeMs.getMillis / 1000L),
```
Comment (dtaniwaki) [inline src/main/scala/com/github/dtaniwaki/akka_pusher/PusherJsonSupport.scala]: Is this related to this PR?

**Comment by dtaniwaki** [comment]
I think Actor's message data type in `TriggerMessage` should be `JsValue` because if it's `String` the serialized data becomes double serialized string. What do you think?

File: src/main/scala/com/github/dtaniwaki/akka_pusher/PusherJsonSupport.scala (hunk 2 above)
Comment (tkqubo) [inline src/main/scala/com/github/dtaniwaki/akka_pusher/PusherJsonSupport.scala]: Oh it's not.  Sorry old modification still remains.  I'll remove this!

File: src/main/scala/com/github/dtaniwaki/akka_pusher/PusherJsonSupport.scala (hunk 1 above)
Comment (tkqubo) [inline src/main/scala/com/github/dtaniwaki/akka_pusher/PusherJsonSupport.scala]: You're right.  This needs to be fixed

**Comment by tkqubo** [comment]
> Actor's message data type in `TriggerMessage` should be `JsValue`

It sounds reasonable.  Thanks for your advice.  I will reimplement this.

**Comment by dtaniwaki** [comment]
Thank you!
